src/main/java/com/db/crud/voting/service/AgendaService.java

```diff
@@ -16,6 +16,4 @@ public interface AgendaService {
     AgendaResponse createAgenda(AgendaRequest agendaRequest);
 
     AddVoteResponse addVote(AddVoteRequest addVoteRequest);
```

Comment:
```suggestion
    VoteResponse addVote(AddVoteRequest addVoteRequest);
```

O Prefixo `Add` não é necessário! Um DTO geralmente é usado para transportar dados, e não para indicar a ação que foi realizada. Ao renomear como VoteResponse, você vai ficar claro que o objeto representa uma resposta relacionada a votos, independentemente da operação (adicionar, atualizar, remover) que será realizada.

Reply:
Não tinha parado pra pensar nisso! Um ótimo feedback!

No commit [Rename Vote DTO for better readibility](https://github.com/AurorinhaBoreal/VotacaoCRUD-BackEnd/pull/37/commits/8d49f198a21a655a5b310d3d6219655ce0adaa90))[https://github.com/AurorinhaBoreal/VotacaoCRUD-BackEnd/pull/37/commits/8d49f198a21a655a5b310d3d6219655ce0adaa90]